storagenode/orders/db_test.go

```diff
@@ -46,7 +47,9 @@ func TestOrders(t *testing.T) {
 		require.NoError(t, err)
 		require.Len(t, emptyArchive, 0)
 
-		now := ptypes.TimestampNow()
+		now := time.Now().UTC()
```

Comment:
One last `UTC()` to get rid of :)
Unless you have a good reason why we have it?